pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -219,6 +219,41 @@ public DataExporter(Config cfg) {
         return activities;
     }
 
+    /**
+     * Find and set the max number of instances seen per participant across the study for each given activity extract.
+     *
+     * @param handle     the database handle
+     * @param activities the list of activities to look at
+     */
+    public void computeMaxInstancesSeen(Handle handle, List<ActivityExtract> activities) {
+        var instanceDao = handle.attach(ActivityInstanceDao.class);
+        for (var activity : activities) {
```

Comment:
Showing the type of activity or activities or both would be helpful here.

Reply:
Two lines above, we have the type annotation there, is that sufficient?

Reply:
Done

Reply:
That  should be fine. Never mind.